distribution/docs/src/main/resources/_contents/_quickstart/quickstart-configuring-contents.adoc

```diff
@@ -0,0 +1,15 @@
+
+=== Configuring (Quick Install)
+
+Set the configurations needed to run ${branding}
+
+.. Go to ${secure_url}/admin.
+.. Enter the default username of `admin` and the password of `admin`.
+.. Follow the prompts installer prompts for a standard installation.
+... Click start to begin the setup process.
+... Configure guest claims attributes or use defaults.
```

Comment:
I say keep defaults, but agree that if we need to configure anything, we should say how. Second opinion, @stustison ?